src/platform_impl/web/event_loop/runner.rs

```diff
@@ -234,64 +234,74 @@ impl Shared {
         ));
 
         let runner = self.clone();
-        let window = self.window().clone();
         *self.0.on_mouse_move.borrow_mut() = Some(EventListenerHandle::new(
             self.window().clone(),
             "pointermove",
-            Closure::new(move |event: PointerEvent| {
-                if !runner.device_events() {
-                    return;
-                }
+            Closure::new({
+                let mut delta = backend::event::MouseDelta::new();
```

Comment:
Hmm, maybe you can keep this out where `let window` was before (around line 237), to reduce the diff noise?

Reply:
Good point, no reason to scope the variable like this anyway.